framework/src/outputs/Console.C

```diff
@@ -366,7 +366,7 @@ Console::output(const ExecFlagType & type)
       _old_nonlinear_norm = std::numeric_limits<Real>::max();
 
     _console << std::setw(2) << _nonlinear_iter
-             << " Nonlinear |R| = " << outputNorm(_old_nonlinear_norm, _norm) << '\n';
+             << " Nonlinear |R| = " << outputNorm(_old_nonlinear_norm, _norm) << std::endl;
```

Comment:
I do not understand your argument about hundreds. You added a flush in each NL and linear iteration. Easily there can be hundreds iterations just within a single NL step.